### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by roseateros**
Changed run command as per the latest python script file name

### CODE REVIEW

**Review by roseateros — COMMENTED** [review]
Changed run script as per file name.

File: README.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ scode	cscode	pref	reason
 
 ## Command
 
-To run `school_center.py` use the command below:
+To run `app.py` use the command below:
 
 ```bash
-python3 school_center.py sample_data/schools_grade12_2081.tsv sample_data/centers_grade12_2081.tsv sample_data/prefs.tsv
+python3 app.py sample_data/schools_grade12_2081.tsv sample_data/centers_grade12_2081.tsv sample_data/prefs.tsv
```
Comment (dhirajraut1) [inline README.md:74]: In order to run the `app.py`, you need to execute 
```bash
streamlit run app.py
```

**Review by dhirajraut1 — COMMENTED** [review]
Better to add a new instruction for `app.py` instead of editing the old one

**Comment by roseateros** [comment]
You’re right.

**Pull request closed without merging**